### DISCUSSION TITLE
該迎合大眾還是做自己

### DISCUSSION THREAD

**Discussion by CPW0919** [1101-GS4719-C/Python-Community-GS4719-C#1318] [General]
如題，這是蠻多領域的創作者都會思考的問題，像有些我以前很喜歡的youtuber或是歌手每當名氣越來越上漲時，往往也會在兩者間作出抉擇，但每次只要看到他們慢慢走向商業化時，就覺得雖然很可惜不能再看到他們更真實的一面，但又替他們開心因為誰不想賺更多錢，所以各位覺得有甚麼兩全的辦法能在保持自我的同時也能取得大眾的喜好呢?

**Comment by Jamesmoneya** [comment]
剛開始都是做做看的心態，因此他們不會把經營網路變成為一的經濟來源，一定會有其他的方式來支撐，但是等到有一點名氣了，想要全職網紅的時候，如果沒有吧利益放在第一，他們就沒飯吃了，我想如果他們要找回之前的感覺，去找一份工作沉澱一下

**Comment by llh26** [comment]
我真的很佩服那些能堅持做自己喜歡的音樂的創作者，雖然背後可能有很多因素使然，其實有聽過很多創作者會先轉換風格賺一些錢之後，再將這些資金投入自己想做的方向，我覺得這也是一個很不錯的選擇！雖然我對創作者的生態不是很了解，但這樣作法也算是找到兩者之間的平衡點，不知道文章作者有什麼其他想法嗎～
